src/main/java/com/flagsmith/models/Flags.java

```diff
@@ -150,8 +151,12 @@ public List<BaseFlag> getAllFlags() {
    * @param featureName Feature name
    * @return
    */
-  public Boolean isFeatureEnabled(String featureName) throws FlagsmithClientError {
-    return flags.containsKey(featureName) ? getFlag(featureName).getEnabled() : null;
+  public boolean isFeatureEnabled(String featureName)
+      throws FlagsmithClientError {
+    if(!this.flags.containsKey(featureName)) {
```

Comment:
We don't need this here, right? It's covered by `getFlag` ?

Reply:
👍🏻